### PR TITLE
fix: (TNLT-6596) revert (TNLT-2130) remove caption under main standard lead imag…

### PR DESCRIPTION
**Pull request by kamiranoff**
- https://nidigitalsolutions.jira.com/browse/TNLT-6596
- revert caption removed under lead asset as per legal requirements

**Before**
Simulator Screen Shot - iPhone 12 Pro - 2021-02-02 at 12 44 02

**After**
Simulator Screen Shot - iPhone 12 Pro - 2021-02-02 at 12 43 14

**iPad**
Simulator Screen Shot - iPad Pro (12 9-inch) (4th generation) - 2021-02-02 at 13 32 08

### CODE REVIEW

File: packages/article-magazine-standard/src/article-magazine-standard.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* eslint-disable consistent-return */
-import React, { Component } from "react";
+
+import React, { Component, Fragment } from "react";
```
Comment (c-j-j) [inline packages/article-magazine-standard/src/article-magazine-standard.js:3]: Do we need `Fragment` imported here?

**Review by c-j-j — COMMENTED** [review]
Could you please share a screenshot for tablet to see how this now looks?

File: packages/article-magazine-standard/src/article-magazine-standard.js (hunk 1 above)
Comment (kamiranoff) [reply packages/article-magazine-standard/src/article-magazine-standard.js:3]: I guess not. I just reverted a PR. but can remove it.

Comment (c-j-j) [reply packages/article-magazine-standard/src/article-magazine-standard.js:3]: Yeah I can't see Fragment or <> in use here - think we can remove

**Comment by kamiranoff** [comment]
> Could you please share a screenshot for tablet to see how this now looks?

done!

**Review by Geoff-Ford — APPROVED** [review]

File: packages/article-magazine-standard/src/article-magazine-standard.js (hunk 1 above)
Comment (c-j-j) [reply packages/article-magazine-standard/src/article-magazine-standard.js]: Also our linter should have caught this - should review eslint settings if it didn't

**Review by c-j-j — APPROVED** [review]

File: packages/article-magazine-standard/src/article-magazine-standard.js (hunk 1 above)
Comment (kamiranoff) [reply packages/article-magazine-standard/src/article-magazine-standard.js]: There are some unused variables imports in the project. Might need some stronger eslint rules indeed.

**Pull request merged**